serverless/src/samlCallback/handler.js

```diff
@@ -25,25 +26,23 @@ const getLaunchpadToken = (cookieString) => {
  */
 const samlCallback = async (event) => {
   const { body, headers } = event
-  const { Cookie } = headers
+  const { Cookie: headerCookie } = headers
   const { mmtHost } = getApplicationConfig()
 
   const params = new URLSearchParams(body)
   const path = params.get('RelayState')
 
-  const launchpadToken = getLaunchpadToken(Cookie)
-  const samlResponse = parseSaml(params.get('SAMLResponse'))
-  const { auid } = samlResponse
+  const launchpadToken = getLaunchpadToken(headerCookie)
 
-  // There appears to be a limitation in AWS to only allow sending 1 cookie, so we are sending
-  // 1 cookie with multiple values in a base 64 encoded json string.
-  let setCookie = `launchpadToken=${launchpadToken}; Secure; Path=/; Domain=.earthdatacloud.nasa.gov`
+  const edlProfile = await fetchEdlProfile(launchpadToken)
 
-  if (process.env.IS_OFFLINE) {
-    setCookie = `launchpadToken=${launchpadToken}; Path=/`
-  }
+  // Create JWT with launchpad token and edl profile
+  const jwt = createJwt(launchpadToken, edlProfile)
+
+  const location = `${mmtHost}/auth-callback?target=${encodeURIComponent(path)}`
 
-  const location = `${mmtHost}/auth_callback?target=${encodeURIComponent(path)}&auid=${encodeURIComponent(auid)}`
+  const { COOKIE_DOMAIN } = process.env
+  const setCookie = `_mmt_jwt=${jwt}; SameSite=Strict; Path=/; Secure; Domain=${COOKIE_DOMAIN}; Max-Age=900`
```

Comment:
Should this `900` be its own ENV variable or use one of the already configured expiration values?

Reply:
Should be the config value